addons/website_sale_referral/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_referral_tracking_public,Referral Tracking Public Access,website_sale_referral.model_referral_tracking,base.group_public,1,0,0,0
+access_referral_tracking_portal,Referral Tracking Portal Access,website_sale_referral.model_referral_tracking,base.group_portal,1,0,0,0
+access_referral_tracking_user,Referral Tracking User Access,website_sale_referral.model_referral_tracking,base.group_user,1,1,0,0
```

Comment:
Do we really want an internal user to modify the tracking ? I would grant this privilege to the group_system only.

Reply:
Done